platform/mellanox/mlnx-platform-api/sonic_platform/fan.py

```diff
@@ -22,17 +23,24 @@
 
 FAN_PATH = "/var/run/hw-management/thermal/"
 LED_PATH = "/var/run/hw-management/led/"
+CONFIG_PATH = "/var/run/hw-management/config"
```

Comment:
i think we have this config path on different files. is this correct to have it this way instead of having it inherited?

Reply:
I think this is correct. The config path here is a directory and I only need it to find some i2c address related to PSU fan. I don't see difference among differenct platforms.